plugins/modules/aci_bd_to_netflow_monitor_policy.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Samita Bhattacharjee (@samitab) <samitab@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_bd_to_netflow_monitor_policy
+short_description: Bind Bridge Domain to Netflow Monitor Policy (fv:RsBDToNetflowMonitorPol)
+description:
+- Bind Bridge Domain to Netflow Monitor Policy on Cisco ACI fabrics.
+options:
+  bd:
+    description:
+    - The name of the Bridge Domain.
+    type: str
+    aliases: [ bd_name, bridge_domain ]
+  tenant:
+    description:
+    - The name of the Tenant.
+    type: str
+    aliases: [ tenant_name ]
+  netflow_monitor_policy:
+    description:
+    - The name of the Netflow Monitor Policy.
+    type: str
+    aliases: [ name, policy, monitor_pol]
```

Comment:
can you change the aliases to `[ netflow_monitor, netflow_monitor_name, name ]` for consistency with the aci_netflow_monitor_policy module?

Reply:
Done!